minecraft/protocol/packet/clientbound_debug_renderer.go

```diff
@@ -0,0 +1,70 @@
+package packet
+
+import (
+	"github.com/go-gl/mathgl/mgl32"
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+const (
+	ClientBoundDebugRendererTypeClear int32 = iota + 1
+	ClientBoundDebugRendererTypeAddCube
+)
+
+// ClientBoundDebugRenderer is sent by the server to spawn an outlined cube on client-side.
+type ClientBoundDebugRenderer struct {
+	// Type is the type of action. It is one of the constants above.
+	Type int32
+	// Text is the text that is displayed above the debug.
+	Text string
+	// Position is the position to spawn the debug on.
+	Position mgl32.Vec3
+	// Red is the red value from the RGBA colour rendered on the debug.
+	Red float32
+	// Green is the green value from the RGBA colour rendered on the debug.
+	Green float32
+	// Blue is the blue value from the RGBA colour rendered on the debug.
+	Blue float32
+	// Alpha is the alpha value from the RGBA colour rendered on the debug.
+	Alpha float32
```

Comment:
Are these 0-1? Might be good to document that.

Reply:
literally it tell us this is ARGB but each channel is store as uint8([reference](https://en.m.wikipedia.org/wiki/RGBA_color_model))

Reply:
Except, as the fields indicate, these are float32s, not uint8s.